src/pybind/mgr/dashboard/frontend/src/app/core/navigation/administration/administration.component.html

```diff
@@ -18,5 +18,9 @@
             *ngIf="configOptPermission.read"
             routerLink="/telemetry"
             i18n>Telemetry configuration</button>
+    <button ngbDropdownItem 
+            *ngIf="configOptPermission.read" 
+            routerLink="/feedback" 
+            i18n>Send Feedback</button>
```

Comment:
How about this or something similar?
```suggestion
            i18n>Raise an Issue</button>

Reply:
Good question. As users may want to both report issues AND send suggestions (are suggestions or improvements "reported?)... Any phrasing that comprises both meanings?

Let's ping an expert: @zdover23, what's your take on this?